### PR TITLE
Clarify the scope of this project

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
cc @mrunalp

**Comment by thockin** [comment]
Right direction.  LGTM

File: README.md [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+
+ocid is meant to provide an integration path between OCI conformant runtimes and the kubelet.
+Specifically, it implements the Kubelet Container Runtime Interface (CRI) using OCI conformant runtimes.
+The scope of OCID is tied to the scope of the CRI.
+
+At a high level, we expect the scope of OCID to be restricted to the following functionalities:
+
+* Support multiple image formats including the existing Docker image format 
+* Support for multiple means to download images including trust & image verification
+* Container Image management (managing image layers, overlay filesystems, etc)
```
Comment (mrunalp) [inline README.md]: nit: Image -> image

File: README.md [hunk 2]
```diff
@@ -1,14 +1,36 @@
 ocid - OCI-based implementation of Kubernetes Container Runtime Interface [![Build Status](https://travis-ci.org/kubernetes-incubator/ocid.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/ocid)
 =
 
+# What is the scope of this project?
+
+ocid is meant to provide an integration path between OCI conformant runtimes and the kubelet.
```
Comment (mrunalp) [inline README.md]: s/ocid/OCID

**Comment by mrunalp** [comment]
@smarterclayton @rhatdan PTAL

**Comment by rhatdan** [comment]
LGTM

**Comment by mikebrow** [comment]
LGTM

File: README.md [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
+* Monitoring and logging required to satisfy the CRI
+* Resource isolation as required by the CRI
+
+
+# What is not in scope for this project?
+
+* Building, signing and pushing images to various image storages
+* A CLI utility for interacting with OCID. Any CLIs built as part of this project are only meant for testing this project and there will be no guarantees on the backwards compatibility with it.
+
 ### Status: pre-alpha
```
Comment (mrunalp) [inline README.md]: This should be under the top header like before.

**Comment by runcom** [comment]
SGTM...

**Comment by mrunalp** [comment]
Looks fine. Just waiting for comments to be addressed. Thanks!

**Comment by vishh** [comment]
Thanks for the quick review everyone. @mrunalp comments addressed. PTAL.

**Comment by mrunalp** [comment]
LGTM. Thanks!

**Pull request merged**